addons/base_vat/models/res_partner.py

```diff
@@ -750,6 +751,16 @@ def check_vat_br(self, vat):
         is_cnpj_valid = stdnum.get_cc_module('br', 'cnpj').is_valid
         return is_cpf_valid(vat) or is_cnpj_valid(vat)
 
+    __check_vat_cr_re = re.compile(r"^\d{8,11}$")
```

Comment:
according to the commit message, shouldn't it be like this?
```suggestion
    __check_vat_cr_re = re.compile(r"^\d{9,12}$")
```

Reply:
Hi @alialfie 

Please check official documentation for EDI documents 

[ANEXOS Y ESTRUCTURAS](https://atv.hacienda.go.cr/ATV/ComprobanteElectronico/docs/esquemas/2016/v4/ANEXOS%20Y%20ESTRUCTURAS.pdf)

#### Page No. 4 

* El contribuyente debe estar inscrito ante la Administración Tributaria.
* Este campo será de condición obligatoria, cuando se posea información en el nodo “Tipo de identificación del emisor.

#### Formato:

* La “Cédula física” debe de contener 9 dígitos, sin cero al inicio y sin guiones.
* La “cédula de personas Jurídicas” debe contener 10 dígitos y sin guiones.
* El “Documento de Identificación Migratorio para Extranjeros ( DIMEX)” debe contener 11 o 12 dígitos, sin ceros al inicio y sin guiones.
* El “Documento de Identificación de la DGT( NITE)” debe contener 10 dígitos y sin guiones.

cc: @luistorresm

Reply:
Hi @randall-vx Sorry for the late reply
Thanks for the docs, so shouldn't the regex match the length to be 9-12 instead of 8-11 then like I suggested above?

Reply:
Hi @alialfie ! 

Correct! The regex should match a length of 9-12 characters instead of 8-11.

cc: @luistorresm